0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -147,61 +145,62 @@ Connectors implement the interledger protocol to forward payments between ledger
 
-#### amount
+The type value `1` indicates an ILP Packet. The type values `2` through `7` indicate ILQP Packets.
 
-    UInt64 ::= INTEGER (0..18446744073709551615)
+#### Life-Cycle of the ILP Packet
 
-Amount in discrete units of the receiving ledger's asset type. Note that the amount is counted in terms of the smallest indivisible unit on the receiving ledger.
+The exact use of the ILP Packet depends on the transport protocol you use. In [IPR][], the receiver generates and communicates the packet to the sender. In [PSK][], the sender generates the packet according to agreed-upon rules with a pre-shared secret.
+[IPR]: ../0011-interledger-payment-request/0011-interledger-payment-request.md
 
-#### account
+When the sender prepares a transfer to start the payment, the sender attaches the ILP Packet to the transfer, in the memo field if possible. If a ledger does not support attaching the entire ILP Packet to a transfer as a memo, users of that ledger can transmit the ILP Packet using another system, but MUST be able to correlate transfers and ILP Packets. ***Rome's note: Doesn't this also have some sort of trust implications, like the system for correlating transfers must be trusted to the same extent as the ledger because it could be used to steal money if it's malicious?***
 
-    -- Readable names for special characters that may appear in ILP addresses
-    hyphen IA5String ::= "-"
-    period IA5String ::= "."
-    underscore IA5String ::= "_"
-    tilde IA5String ::= "~"
+When a connector sees an incoming prepared transfer with an ILP Packet, the connector reads the ILP Packet to get the ILP Address of the payment's receiver. If the connector has a route to the receiver's account, the connector prepares a transfer to continue the payment, and attaches the same ILP Packet to the new transfer.
 
-    -- A standard interledger address
-    Address ::= IA5String
-      (FROM
-        ( hyphen
-        | period
-        | "0".."9"
-        | "A".."Z"
-        | underscore
-        | "a".."z"
-        | tilde )
-      )
-      (SIZE (1..1023))
+When the receiver sees the incoming prepared transfer, the receiver reads the ILP Packet to confirm the details of the packet. The receiver confirms that the amount from the ILP Packet matches the amount actually delivered by the transfer. The receiver decodes the data of the ILP Packet and matches the condition to the packet. (Depending on the transport protocol, the packet's data MAY contain the condition.) The receiver MUST confirm the integrity of the ILP Packet, for example with a [hash-based message authentication code (HMAC)](https://en.wikipedia.org/wiki/Hash-based_message_authentication_code). If the receiver finds the transfer acceptable, the receiver releases the fulfillment for the transfer, which can be used to execute all the prepared transfers.
```

Comment:
> (Depending on the transport protocol, the packet's data MAY contain the condition.)

We decided a while back that this doesn't make sense, because it's redundant and provides no additional value. Instead, it's more likely that the condition includes the MAC of the packet.